web_client/js/imageViewerSelectWidget.js

```diff
@@ -34,6 +34,11 @@ girder.views.ImageViewerSelectWidget = girder.View.extend({
                 type: girder.views.OpenseadragonImageViewerWidget
             },
             {
+                name: 'slideatlas',
```

Comment:
There's a duplicate commented-out version of this at line 56. Can you please delete one of them?